pkg/cloud/services/networking/network.go

```diff
@@ -375,3 +353,11 @@ func GetSubnetsByFilter(networkClient *gophercloud.ServiceClient, opts subnets.L
 	}
 	return snets, nil
 }
+
+func getSubnetName(clusterName string) string {
```

Comment:
those 2 function are identical , on purpose?

Reply:
Both getSubnetName and getNetworkName return the same string.  This is intentional. getSubnetName is used for getting subnet name. getNetworkName is used for getting network name.